test/src/org/labkey/test/pages/cds/MAbDataGrid.java

```diff
@@ -201,6 +204,83 @@ public String getMabCellValue(String mabName, String columnLabel)
         return cells.get(colInd).getText();
     }
 
+    public void clearAllSelections()
+    {
+        Locator.XPathLocator checkbox = Locators.headerCheckboxLoc;
+        Locator.XPathLocator checkedLoc = checkbox.withClass("x-grid-hd-checker-on");
+        if(_webDriverWrapper.isElementPresent(checkedLoc))
+            _webDriverWrapper.click(checkbox);
+    }
+
+    public void selectMAbs(String ...mabNames)
+    {
+        checkMAbs(true, mabNames);
+    }
+
+    public void unselectMAbs(String ...mabNames)
+    {
+        checkMAbs(false, mabNames);
+    }
+
+    public void checkMAbs(boolean check, String ...mabNames)
+    {
+        for (String mabName: mabNames)
+        {
+            if (isMabChecked(mabName) != check)
+            {
+                _webDriverWrapper.click(Locators.getMabCheckbox(mabName));
+            }
+        }
+    }
+
+    public boolean isMabChecked(String mabName)
+    {
+        return _webDriverWrapper.isElementPresent(Locators.getSelectedRowLocByMabName(mabName));
+    }
+
+    public void openDilutionReport()
+    {
+        openMAbReport(NAB_MAB_DILUTION_REPORT);
+    }
+
+    public void openIC50Report()
+    {
+        openMAbReport(NAB_MAB_IC50_REPORT);
+    }
+
+    public void openMAbReport(String reportName)
+    {
+        _webDriverWrapper.click(Locators.getMAbReportBtn(reportName));
+        _webDriverWrapper.sleep(500);
+        _webDriverWrapper.waitForElement(Locators.getMAbReportHeader(reportName));
```

Comment:
If you would rather you could try removing the sleep on the line before and add:
WebDriverWrapper.WAIT_FOR_JAVASCRIPT +  500 
as the second parameter to the waitForElement.